Floofbot/Services/Repository/Models/FilteredWord.cs

```diff
@@ -5,7 +5,7 @@ namespace Floofbot.Services.Repository.Models
     public partial class FilteredWord
     {
         [Key]
-        public ulong Id { get; set; }
+        public int Id { get; set; }
```

Comment:
why downgrade to an int?

Reply:
sqlite does not support ordering by ulong